content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+
+Our use of third-party service providers includes:
+- GitHub - to host and contribute to projects
+- Mailchimp - store an email list for marketing purposes
+- Slack - for community communication
+- Eventbrite???
+
+If you have any questions about the specific service providers we currently use, please contact us at questions@innersourcecommons.org or by using the contact information set out at the bottom.
+
+Legal Compliance / Protection of the Public and Our Business / Legitimate Interests 
+
+We will release personal and account information in response to lawful requests by public authorities, including to meet legitimate national security or law enforcement requirements; to protect, establish, or exercise our legal rights or defend against legal claims, including to collect a debt; to comply with a subpoena, court order, legal process, or other legal requirement; or when we believe in good faith that such disclosure is necessary to comply with the law, prevent imminent physical harm or financial loss, or investigate, prevent, or take action regarding illegal activities, suspected fraud and threats to our property.
+
+In limited circumstances, we may also exchange certain information with other companies and organizations for the specific purpose of fraud protection. However, this does not include selling, renting, sharing, or otherwise disclosing personally identifiable information from you for commercial purposes in violation of the commitments set forth here.
+
+We may also share your information during an organizational transaction like a merger or sale of our assets. If such a transaction occurs, we will provide notification of any changes to control of your information, as well as choices you may have.
+
+## Children’s Privacy
+
+Our services are not designed for, and are not marketed to, people under the age of 18 (“minors”). We do not knowingly collect or ask for information from minors and we do not knowingly allow minors to use our services. By using our services or accessing our website, you represent that you are at least the age of majority in your country, state and/or province of residence.
+
+## Data Transfers
+
+InnerSource Commons is based in the United States, processes and stores data in the United States, and makes its services available around the world. The United States, Member States of the European Economic Area (“EEA”), and other countries are governed by different laws. When your data is moved from its home country to another country, the laws and rules that protect your personal information in the country to which your information is transferred may be different from those in the country where you reside. For example, the legal requirements for law enforcement to gain access to personal information may vary between countries. If your personal data is in the United States, it may be accessed by government authorities in accordance with United States law.
```

Comment:
> InnerSource Commons is based in the United States

This is only for the foundation? Is the site represents the foundation? 
BTW, Github Pages -- where the site is actually hosted based in the US

Reply:
Yes - this should be InnerSource Commons Foundation when referring to the legal entity. And yes, the site is the responsibility of the Foundation.